opensrp-anc/src/main/java/org/smartregister/anc/library/util/JsonFormUtils.java

```diff
@@ -127,12 +128,70 @@ public static JSONObject getFormAsJson(JSONObject form, String formName, String
                 form.put(JsonFormUtils.ENTITY_ID, entityId);
             }
         } else {
-            Timber.tag(TAG).w("Unsupported form requested for launch " + formName);
+            Timber.tag(TAG).w("Unsupported form requested for launch %s", formName);
         }
-        Timber.d("form is " + form.toString());
+        Timber.d("form is %s", form.toString());
         return form;
     }
 
+    private static void addRegLocHierarchyQuestions(@NonNull JSONObject form) {
+        try {
+            JSONArray fields = form.getJSONObject(JsonFormConstants.STEP1).getJSONArray(JsonFormConstants.FIELDS);
```

Comment:
What happens in the the location questions are not on `STEP1`? Some registration forms are multi stepped.